CONTRIBUTING.adoc

```diff
@@ -15,10 +15,10 @@ Or if you are interested in development, start with:
 
 * <<openshift-development>>
 
-And choose between:
+	And choose between:
```

Comment:
this indent is making markdown interpret it as a code/preformatted blog.  i'm not a markdown guru but you'll need to find another way to do this.